beacon/Cargo.toml

```diff
@@ -10,8 +10,8 @@ base64 = ">=0.21"
 byteorder = "1"
 helium-proto = { path = "../", features = ["services"] }
 prost = {  workspace = true }
-rand = ">=0.8"
-rand_chacha = ">=0.3"
+rand = "<=0.8.5"
```

Comment:
That’s too tight. =0.8

Reply:
@madninja so you want it pinned at 0.8 ?

Reply:
< 0.9